bootcode/bootcode.js

```diff
@@ -0,0 +1,26 @@
+(function (g) {
+    function toB(v) {
+        if (typeof v != 'string') {
+            v = new Uint8Array(v);
+        }
+        return v;
+    }
+    g.pink = {
+        invokeContract: (c) => {
+            return __hostCall(0, {
+                callee: toB(c.callee),
+                selector: c.selector,
+                input: toB(c.input),
+                gasLimit: c.gasLimit || 0,
+                value: c.value || 0,
```

Comment:
Better do simple type validation with `typeof`, and `isNaN`:

```javascript
function isValidNumeric(value) {
    return (typeof value === 'number' && !isNaN(value) || (typeof value === 'bigint')
}
```

Reply:
The QuickJS's C API itself would do checks. It would fail to get values if it is invalid. I want to keep the boot code as thin as possible to reduce the engine code size and to be faster (The engine is running in ink which is really slow).

Reply:
OK I'm got your point.